prime-angular-frontend/src/app/modules/health-auth-site-reg/shared/services/authorized-user.service.ts

```diff
@@ -38,7 +37,7 @@ export class AuthorizedUserService implements IAuthorizedUserService {
   }
 
   public get authorizedUser$(): Observable<AuthorizedUser | null> {
-    // Allow subscriptions, but make immutable
+    // Allow subscriptions, but make the subject immutable
```

Comment:
Same as above, subject or object?

Reply:
No, it's a BehaviourSubject being made immutable